### PR TITLE
bnc#965311 - yast ldap-server-ssl-check does not handle ldaps only ldap+starttls works

### PR DESCRIPTION
**Pull request by varkoly**
https://bugzilla.suse.com/show_bug.cgi?id=965311

### CODE REVIEW

File: tools/ldap-server-ssl-check.cpp [hunk 1]
```diff
@@ -10,11 +10,19 @@ int main(int argc, char** argv)
...
-		lc.start_tls();
-	}
+        try
+        {
+                LDAPConnection lc( uri );
+         try {
+             lc.bind();
+             exit(0);
+         } catch (LDAPException e) {
+                if (e.getResultCode() == 49) {
```
Comment (mvidner) [inline tools/ldap-server-ssl-check.cpp]: Why not 42?

File: tools/ldap-server-ssl-check.cpp [hunk 2]
```diff
@@ -10,11 +10,19 @@ int main(int argc, char** argv)
...
 	setenv("LDAPTLS_CACERT", argv[2], 1);
-	try 
-	{
-		LDAPConnection lc( uri );
-		lc.start_tls();
-	}
+        try
+        {
+                LDAPConnection lc( uri );
+         try {
```
Comment (mvidner) [inline tools/ldap-server-ssl-check.cpp]: The indentation is wrong, please fix it.

File: tools/ldap-server-ssl-check.cpp (hunk 1 above)
Comment (varkoly) [inline tools/ldap-server-ssl-check.cpp]: There is no error code 42. Error codes between 37 and 47 are not used.

Comment (mvidner) [inline tools/ldap-server-ssl-check.cpp]: 42, 37, 49, 47 are meaningless numbers. `INVALID_CREDENTIALS` is what makes code understandable.
```
/usr/include/LDAPResult.h:        static const int INVALID_CREDENTIALS            = 49;
```

**Comment by mvidner** [comment]
That's strange, have you reverted the changes by accident? GitHub shows a nearly empty diff.

File: Rakefile [hunk 3]
```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp1
```
Comment (jreidinger) [inline Rakefile]: this is wrong. It is commit only for SLE12SP1 maintenance branch. Please revert this change.

**Comment by teclator** [comment]
@varkoly, replacing this PR with #34
